web/website/content/posts/2023-03-14-pi-day.md

```diff
@@ -0,0 +1,370 @@
+# Calculating the digits of Pi in DuckDB with PRQL
+
+## Background
+
+Last week saw the 0.6 release of [PRQL](www.prql-lang.org) which brought with it
+the capability to express Recursive CTEs in PRQL. "Recursive" CTEs aren't
+actually truly recursive in the sense that that term is usually used, rather
+they use a "recursive" (i.e. self-referential) syntax to provide a looping
+construct in SQL.
+
+[PRQL](www.prql-lang.org) is a modern, functional query language for
+transforming data. One of its goals is to simplify working with data wherever
+you can currently use SQL. As such it transpiles to SQL while making available
```

Comment:
```suggestion
you can currently use SQL. As such it compiles to SQL while making available
```
we now use this term (but fine to reject ofc)

Reply:
Thanks